### PR TITLE
Show selector with auto width in displayShow

### PR DESCRIPTION
**Pull request by fernandog**
Implements https://github.com/pymedusa/Medusa/issues/2205

Before:
image

After:
image

### CODE REVIEW

**Comment by p0psicles** [comment]
Screenees?

**Comment by fernandog** [comment]
@p0psicles updated PR body

File: views/partials/showSelector.mako [hunk 1]
```diff
@@ -1,8 +1,8 @@
-<div id="showSelector" class="hidden-print col-md-4 col-sm-12 col-xs-12 pull-md-left" style="margin-left:-21px">
+<div id="showSelector" class="hidden-print col-md-2 col-sm-12 col-xs-12 pull-md-left" style="margin-left:-21px">
     <div class="form-inline">
             <div>
                 <div class="select-show-group pull-left top-5 bottom-5">
-                    <select id="select-show" class="form-control input-sm-custom" style="height:25px;padding:1px">
+                    <select id="select-show" class="form-control input-sm-custom" style="width:auto;height:25px;padding:1px">
```
Comment (p0psicles) [inline views/partials/showSelector.mako]: Can you move this to style.css while your at it?

**Comment by p0psicles** [comment]
You tested on all screen sizes?

**Comment by fernandog** [comment]
@p0psicles I resized browser window and looks good while doing it. Also tested on mobile

File: static/css/style.css [hunk 2]
```diff
@@ -1116,6 +1116,12 @@ div#container-display-show {
     width: 300px;
 }
 
+#show-selector {
+    width: auto;
+    height:25px;
+    padding:1px;
+}
+
```
Comment (fernandog) [inline static/css/style.css]: @p0psicles didn't have the same effect

image

Comment (sharkykh) [inline static/css/style.css]: Probably getting overridden by bootstrap.
Try using `!important`, even though I read that it's unrecommended.

Comment (sharkykh) [inline static/css/style.css]: Correction:
You used `#show-selector` instead of `.show-selector`.

Comment (fernandog) [inline static/css/style.css]: @sharkykh that didn't work too.  Can you try? Feel free to commit

Comment (p0psicles) [inline static/css/style.css]: you added the id `show-selector`. that doesn't exist. You should modify the existing `select#select-show`. then it works fine.

**Comment by p0psicles** [comment]
Why would you want to get rid of the 100% width? As currently it's filling out the container (div). So when your viewing on smaller sizes, it wraps to a new line, as how you can see it. You can make the select smaller, but that won't have any effect on the wrapping.

**Comment by sharkykh** [comment]
@p0psicles You can also do `width: 100%;` within a `@media max-width: [insert width]px` (or something like that`

**Comment by p0psicles** [comment]
@fernandog check it out.
I removed the bootstrap columns. And made it wrap to next line, using width: auto.
image

I made some other changes to make sure the show-selector looks good on narrow screen widths.

**Pull request merged**